apps/web/src/views/Ifos/hooks/v3/useGetPublicIfoData.ts

```diff
@@ -1,17 +1,17 @@
+import { Ifo, IfoStatus } from '@pancakeswap/ifos'
+import { bscTokens } from '@pancakeswap/tokens'
 import BigNumber from 'bignumber.js'
-import { useState, useCallback } from 'react'
 import { BSC_BLOCK_TIME } from 'config'
-import round from 'lodash/round'
 import { ifoV2ABI } from 'config/abi/ifoV2'
 import { ifoV3ABI } from 'config/abi/ifoV3'
-import { bscTokens } from '@pancakeswap/tokens'
-import { Ifo, IfoStatus } from '@pancakeswap/ifos'
+import round from 'lodash/round'
```

Comment:
We could even get rid of this import, but well… ok this is not in the scope of this PR 😁